public/javascripts/index.js

```diff
@@ -350,7 +353,7 @@ $(() => {
                             <i class="fa fa-chevron-${toggledRows[order.id] ? "up" : "down"}" aria-hidden="true"/>
                         </button>
                     </td>
-                    <td class="text-right">${order.roomNumber || ""}</td>
+                    <td class="text-right">${order.roomNumber == 'undefined' ? '100': order.roomNumber }</td>
```

Comment:
This is checking whether `order.roomNumber` is the string `"undefined"`. I don't believe that is what you meant. Maybe you wanted to check whether `order.roomNumber` is `undefined` (i.e., not set or was set to `undefined`).
```suggestion
                    <td class="text-right">${order.roomNumber || "100"}</td>
```
In this case, if `order.roomNumber` is falsy (such as undefined or empty string), then it defaults to `"100"`.